lib/senkyoshi/models/outcome_definition.rb

```diff
@@ -0,0 +1,51 @@
+require "senkyoshi/models/resource"
+require "senkyoshi/models/assignment_group"
+
+module Senkyoshi
+  class OutcomeDefinition < Resource
+    attr_reader :id, :content_id, :asidataid, :is_user_created
+    def self.from(xml, category)
+      outcome_definition = OutcomeDefinition.new(category)
+      outcome_definition.iterate_xml(xml)
+    end
+
+    def initialize(category)
+      @category = category
+    end
+
+    def iterate_xml(xml)
+      @content_id = xml.xpath("./CONTENTID/@value").text
+      @asidataid = xml.xpath("./ASIDATAID/@value").text
+      @id = xml.xpath("./@id").text
+      @title = xml.xpath("./TITLE/@value").text
+      @points_possible = xml.xpath("./POINTSPOSSIBLE/@value").text
+      @is_user_created = xml.xpath("./ISUSERCREATED/@value").text == "true"
```

Comment:
You will need to add `include Senkyoshi` to this class, but there is a method that does this check for you.

```ruby
true? xml.xpath("./ISUSERCREATED/@value").text
```